Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerConfigurationCocoa.mm

```diff
@@ -105,6 +105,25 @@
     return m_declarativeNetRequestStoreDirectory;
 }
 
+String WebExtensionControllerConfiguration::registeredContentScriptsDirectory()
```

Comment:
This needs to be elsewhere, since it is per Context. You will want to put this on WebExtensionContext, and use that per-extension subdirectory.